trappy/ftrace.py

```diff
@@ -273,6 +319,14 @@ def __get_trace_class(self, line, cls_word):
                     return trace_class
         return trace_class
 
+    def __before_window(self, timestamp):
+        return (timestamp < self.window[0] + self.basetime) or \
+               (timestamp < self.abs_window[0])
```

Comment:
Sorry, too lazy to figure this out myself: can this be simplified using the new `max_window` thing?

Reply:
So at first the code was:

```
if (timestamp < self.window[0] + self.basetime) or \
(timestamp < self.abs_window[0]):
    self.lines += 1
    continue

if (self.window[1] and timestamp > self.window[1] + self.basetime) or \
    (self.abs_window[1] and timestamp > self.abs_window[1]):
    return
```

I then turned each `if` condition in a method (`__before_window` and `__after_window`) in the **Methodify window inclusion check** commit. These methods are used in the computation of `max_window`, but it still makes sense to me to keep them as they are used in `__populate_data` for two different checks.

Reply:
Wait no, the `__{before/after}_window` methods are no longer used in the computation of `max_window`, that was in a previous version.